File: src/main/java/com/misakamikoto/layout/contents/service/ContentsService.java
package com.misakamikoto.layout.contents.service;

import com.misakamikoto.layout.contents.mapper.ContentsMapper;
import com.misakamikoto.layout.contents.model.ContentsVO;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.io.IOException;
import java.util.List;

/**
 * Created by Misaka on 2016-03-16.
 */
@Service
public class ContentsService {
    /**
     * The Contents mapper.
     */
    @Autowired
    private ContentsMapper contentsMapper;

    /**
     * Gets newest content.
     *
     * @return the newest content
     */
    public ContentsVO getNewestContent() {
        return this.contentsMapper.getNewestContent();
    }

    public List<ContentsVO> getContentsList(int codeId) throws IOException {
        return this.contentsMapper.getContentsList(codeId);
    }
}

File: src/main/webapp/js/service/layout/contentsService.js
/**
 * Created by misaka_mikoto on 2017. 12. 13..
 */
app.service('ContentsService', ['$log', 'AjaxService',  function ($log, AjaxService) {
    this.loadList = function(codeId) {
        return AjaxService.callGet('/contents/list/' + codeId).then(function (data) {
            return data;
        });
    };
}]);